### ISSUE TITLE
Fingerprint scanner A13

### ISSUE THREAD

**Issue by ufos97** [fakemanoan/LineageOS-Releases#43]
Fingerprint sensor not scanning. It's detecting the finger when placed to record print but it's not scanning

**Comment by fakemanoan** [comment]
The fingerprint scanner does work, however I believe the issue lies with the scanner remaining active even when the screen is off. This results in the fingerprint scanner triggering even when its not supposed to and timing out the fingerprint because of "too many wrong attempts"

Could you see if this is true by firstly getting the issue you are describing, then unlocking the phone. Then lock the phone and try the fingerprint reader again. If I am right the fingerprint scanner should work. 

Hope that makes sense

**Comment by ufos97** [comment]
I wasn't able to register fingerprint at all after setting up my phone. So I use only password. When I try registering print that's when I get the issue. Thanks

**Comment by POSSYGEE** [comment]
Is there beta a13 version for g925f ?

**Comment by hamzamaqsood877** [comment]
I dirty flashed A13 (as I did back from A11 to A12), so this fingerprints were already saved. I tried to add another one but it went back as I put my finger on sensor. I tried again and scanning option was not highlighted.
![Screenshot_20230519-003304_Settings.png](https://github.com/fakemanoan/LineageOS-Releases/assets/73820858/bb13f142-2552-4805-8149-078adb599076)

While fingerprint senson while unlocking the phone is working fine. And you're right, it's working even while screen is off. 👍
